File: Web_veguita/Negocio/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Negocio
{
    public class Usuario
    {
        public int IdUsuario { get; set; }
        public String Contraseña { get; set; }

        public Usuario()
        {
            IdUsuario = 0;
            Contraseña = string.Empty;
        }

        public Usuario(int parId, String parContraseña)
        {
            IdUsuario = parId;
            Contraseña = parContraseña;
        }
    }
}

File: AplicacionPoS/src/negocio/Usuario.java
package negocio;

public class Usuario {
    private int id;
    private String contraseña;

    public Usuario() {
        this.id = 0;
        this.contraseña = new String();
    }
    
    public Usuario(int id, String contraseña) {
        this.id = id;
        this.contraseña = contraseña;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getContraseña() {
        return contraseña;
    }

    public void setContraseña(String contraseña) {
        this.contraseña = contraseña;
    }
    
    
}
